pkg/kubelet/cri/streaming/remotecommand/httpstream.go

```diff
@@ -155,7 +155,7 @@ func createHTTPStreamStreams(req *http.Request, w http.ResponseWriter, opts *Opt
 	case remotecommandconsts.StreamProtocolV2Name:
 		handler = &v2ProtocolHandler{}
 	case "":
-		klog.V(4).Infof("Client did not request protocol negotiation. Falling back to %q", remotecommandconsts.StreamProtocolV1Name)
+		klog.V(4).InfoS("Client did not request protocol negotiation. Falling back to channel.k8s.io", "protocol", remotecommandconsts.StreamProtocolV1Name)
```

Comment:
Since this label pair has a constant value I don't think we should use it as a key/value pair.

Probably better to format it into a single string. @serathius wdyt?

Reply:
Maybe drop the copied constant `channel.k8s.io` ?

Reply:
I have fixed it,thanks.